src/audio/kpb.c

```diff
@@ -57,7 +57,7 @@ struct comp_data {
 	bool is_internal_buffer_full;
 	size_t buffered_data;
```

Comment:
commit msg, language: meaningfull -> meaningful

Patch itself is good. kpb_buffer was odd (key phrase buffer _ buffer)